### PR TITLE
#112 fix colision between efcd006 & cdb1ada commits

### PR DESCRIPTION
**Pull request by CAMOBAP795**
- #112

### CODE REVIEW

File: spec/metanorma_spec.rb [hunk 1]
```diff
@@ -169,10 +169,3 @@ def ruby(x)
   command "metanorma -v -t iso"
   its(:stdout) { is_expected.to match(/Metanorma::ISO \d/) }
 end
-
-RSpec.describe "gives version Metanorma-Cli" do
```
Comment (ronaldtse) [inline spec/metanorma_spec.rb]: @CAMOBAP795 why are we removing this test?

Comment (CAMOBAP795) [reply spec/metanorma_spec.rb]: good catch, I decided to move it to version_spec but forgot to commit it, now it's done

Comment (ronaldtse) [reply spec/metanorma_spec.rb]: Excellent, thanks!

File: spec/metanorma_spec.rb [hunk 2]
```diff
@@ -168,11 +168,4 @@ def ruby(x)
...
   its(:stdout) { is_expected.to match(/Metanorma::ISO \d/) }
-end
-
-RSpec.describe "gives version Metanorma-Cli" do
-  command "metanorma version"
-  its(:stdout) {
-    is_expected.to match(/Metanorma::Cli \d/)
-    is_expected.to match(/Metanorma \d/) }
-end
+end
```
[the remark this answers is not in the document]
Comment (CAMOBAP795) [reply spec/metanorma_spec.rb]: Fixed

**Pull request merged**